app/scopes.json

```diff
@@ -0,0 +1,1639 @@
+[
```

Comment:
bit of a nit-pick, but could we move this to `app/scripts/api-manifest`? it's technically not part of the manifest, but it seems like that would be a good place for it to live instead of root of the `app` folder.